picard/file.py

```diff
@@ -760,6 +761,11 @@ def can_view_info(self):
         return True
 
     def _info(self, metadata, file):
+        try:
+            size = os.path.getsize(encode_filename(self.filename))
+            metadata['~filesize'] = size
```

Comment:
`size` is not needed.

```python
 metadata['~filesize'] = os.path.getsize(encode_filename(self.filename))
```

Reply:
Thank you for mentioning, I will change that...